src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -96,6 +97,23 @@ static void do_unary_operation(cldnn::topology& topology,
     topology.add(cldnn_unary);
 }
 
+static void do_backprop_operation(cldnn::topology& topology,
```

Comment:
Is this function necessary in case of no one else use it? It is ok if you plan to use it in future. If not, place the code in function call location.